main.cpp

```diff
@@ -83,68 +116,112 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+float rentalPrice(unsigned int rentalDuration, unsigned int carType = 0, unsigned int insurance = 0)
+
+{
+    ignoreUnused(rentalDuration, carType, insurance);  
 
+    return {};
+}
 /*
  2)
  */
+int carReference(int carColor = 0, int carLicenseNumber = 00000, unsigned int carConstructionYear = 2015)
```

Comment:
- [ ] don't write `00000` as a default value for an integer.
You probably want a `std::string` here but those haven't been covered yet so do something else, like `int manufacturer = `